boot/bootutil/src/loader.c

```diff
@@ -2137,10 +2137,17 @@ context_boot_go(struct boot_loader_state *state, struct boot_rsp *rsp)
      * have finished. By the end of the loop each image in the primary slot will
      * have been re-validated.
      */
+    volatile int fih_cnt;
```

Comment:
Variables are usually declared at the beginning of a function. Please move it there to be aligned with the existing pattern.